_posts/2019-01-18-monorepos-aws-codebuild.markdown

```diff
@@ -7,15 +7,15 @@ author: Matt Tyler
 
-Much has been written recently on the monorepo, both [for](https://medium.com/@mattklein123/monorepos-please-dont-e9a279be011b) and [against](https://medium.com/@adamhjk/monorepo-please-do-3657e08a4b70). One thing that often isn't covered is how package management plays a role in what you might choose. I live and work outside of the Silicon Valley bubble - working inside of companies with sub-optimal development processes is normal for me, and I would not be working there if everything was sunshine and rainbows. This project, for instance, was located in an enterprise company that would both not want to publish code to the public NPM repository, and at present did not have a hosted package management solution like JFrog Artifactory or Sonatype Nexus. My personal experience is that not an uncommon situation within enterprise companies and smaller shops working with legacy tooling, or with developers with little open source experience. In the latter, it's usually not too difficult to quickly deploy a hosted package management server. This can be a bit more difficult in larger organisations, as a business case needs to be carefully prepared and approved, and then it may take some time for procurement and installation. When this occurs, not using a monorepo is a luxury that you can't afford. With this in mind, I began to look at how we could improve the existing monorepo and its interactions with AWS CodeBuild.
+Much has been written recently on the monorepo, both [for](https://medium.com/@mattklein123/monorepos-please-dont-e9a279be011b) and [against](https://medium.com/@adamhjk/monorepo-please-do-3657e08a4b70). One thing that often isn't covered is how package management plays a role in what you might choose. I live and work outside of the Silicon Valley bubble - working inside of companies with sub-optimal development processes is normal for me, and I would not be working there if everything was sunshine and rainbows. This project, for instance, was located in an enterprise company that neither wanted to publish code to the public NPM repository, nor had a hosted package management solution like JFrog Artifactory or Sonatype Nexus. In my personal experience this is not an uncommon situation within enterprise companies and smaller shops working with legacy tooling, or with developers with little open source experience. In the latter, it's usually not too difficult to quickly deploy a hosted package management server. This can be a bit more difficult in larger organisations, as a business case needs to be carefully prepared and approved, and then it may take some time for procurement and installation. When this occurs, not using a monorepo is a luxury that you can't afford. With this in mind, I began to look at how we could improve the existing monorepo and its interactions with AWS CodeBuild.
```

Comment:
Looks like the for and against links need to be swapped. for has `please-dont` and against has `please-do`.